fMRIVolume/scripts/OneStepResampling.sh

```diff
@@ -135,6 +135,22 @@ JacobianOut=`getopt1 "--ojacobian" $@`  # "${18}"
 fMRIReferencePath=`getopt1 "--fmrirefpath" $@` # "${19}"
 fMRIReferenceReg=`getopt1 "--fmrirefreg" $@`  # "${20}"
 
+#hidden: toggle for unreleased new resampling command, default off
```

Comment:
Maybe add a comment explaining the advantage of the `--wb-resample` approach?

Reply:
I added a comment under the option parsing lines - I expect when workbench is released, we will just remove the applywarp code path.

Reply:
At least initially, I would suggest we leave the applywarp code path, but make `useWbResample=1` the default instead.